tests/test_converter.py

```diff
@@ -6,18 +6,46 @@
     Strikeout, Code, CodeBlock, BulletList, OrderedList, Decimal, Period, Meta, Pandoc, Link, \
     HorizontalRule, BlockQuote, Image, MetaString, Table, TableHead, TableBody, \
     TableFoot, RowHeadColumns, Row, Cell, RowSpan, ColSpan, ColWidthDefault, AlignDefault, \
-    Caption, Math, InlineMath, DisplayMath
+    Caption, Math, InlineMath, DisplayMath, Underline
 
 from n2y import converter, notion
 
-default_annotation = {"bold": False, "italic": False, "strikethrough": False,
-                      "underline": False, "code": False, "color": "default"}
+default_annotation = {
+    "bold": False, "italic": False, "strikethrough": False,
+    "underline": False, "code": False, "color": "default"}
 
 eq1 = "{\\displaystyle i\\hbar {\\frac {d}{dt}}\\vert "
 eq2 = "\\Psi (t)\\rangle={\\hat {H}}\\vert \\Psi (t)\\rangle}"
 default_equation = f"{eq1}{eq2}"
 
 
+def generate_annotated_obj(arr):
+    obj = {
+        'object': 'block',
+        'has_children': False,
+        'archived': False,
+        'type': 'paragraph',
+        'paragraph': {
+            'color': 'default',
+            'text': []}}
+    for block in arr:
```

Comment:
You can expand the tuple in the for loop directly!

Also, `arr` is not a very good name. I'd change it to something else, like `text_blocks_descriptors`:

```suggestion
    for (text, annotations) in text_blocks_descriptors:
```